cmd/podman/system/reset.go

```diff
@@ -44,6 +47,18 @@ func init() {
 }
 
 func reset(cmd *cobra.Command, args []string) {
+	remoteSocket := &define.RemoteSocket{}
+	isLocalSocket := false
+	info, err := registry.ContainerEngine().Info(registry.Context())
+	if err != nil {
+		logrus.Error(err)
+	} else {
+		remoteSocket = info.Host.RemoteSocket
+		path, _ := strings.CutPrefix(remoteSocket.Path, "unix://")
```

Comment:
nit: since you're discarding the returned bool:
```suggestion
		path := strings.TrimPrefix(remoteSocket.Path, "unix://")
```